src/ImageMagick.jl

```diff
@@ -98,11 +80,11 @@ function load_(file::Union{AbstractString,IO,Vector{UInt8}}; ImageType=Image, ex
         cs = "Gray"
     end
 
-    depth = getimagechanneldepth(wand, DefaultChannels)
+    depth = getimagedepth(wand)
     if depth <= 8
         T = UFixed8     # always use 8-bit for 8-bit and less
     else
-        T = ufixedtype[2*((depth+1)>>1)]  # always use an even # of bits (see issue 242#issuecomment-68845157)
+        T = UFixed{UInt16,((depth+1)>>1)<<1}  # always use an even # of bits (see issue 242#issuecomment-68845157)
```

Comment:
supporting >16-bit depths would be easier i think by changing the existing `else` clause to an `elseif depth <= 16` and then adding logic for `UFixed{UInt32,..}` in a subsequent `else` clause.  i figured we'd do that when the time came, but we could add that now if you want.  i don't know whether ImageMagick currently supports it or not.

moreover, the dict uses notation that will be deprecated by https://github.com/JuliaMath/FixedPointNumbers.jl/pull/51.